upp-concept-publishing-provisioner/README.md

```diff
@@ -1,13 +1,52 @@
-# upp-concept-publishing-provisioner
+# AWS Concept Publishing provisioner
 
-Cloudoformation to spin up an S3 bucket, SNS Topic and 1/2 SQS Queues to be used as part of the event-driven concept publishing pipeline.  
+The AWS Concept Publishing provisioning process will:
 
-## Provisioning
-The script can be used directly in the AWS Console.  You need to provide two parameters:
-- Environment tag - Which environment this belongs to.
-- IsMultiRegion - Whether the stack needs to be read in multiple regions (will spin up 2 SQS queues rather than 1).
+* Create an S3 Bucket, SNS Topic and 1/2 SQS Queues using the specified Cloudformation Template which is used as a part of the event-driven concept publishing pipeline.
 
-## Decomissioning
-You can decom the stack by deleting it from the AWS Console.  If the bucket has content in, the stack will fail to delete.  In this case, you should do one of the following:
-- If you don't need the bucket, manually delete it and then re-delete the stack.  It should successfully work.
-- If you want to keep the bucket, re-delete the stack and it should prompt you whether you want to ignore the bucket.  Say yes and the stack should delete leaving the bucket alone.
+The decommissioning process will:
+
+* Delete the S3 Bucket if empty
+* Delete the objects and the S3 bucket if not empty
```

Comment:
Both of these have the same result, might as well get rid of one to make it clearer :)